src/index.html

```diff
@@ -5,8 +5,9 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="./assets/scss/main.scss">
```

Comment:
I would recommend linking to your dist/css/main.css file instead of the main.scss file. This is because the browser can't read scss styles, and all your scss files will be compiled to the main.css file anyway